### PR TITLE
Improvements for windows

### PR DESCRIPTION
**Pull request by juanjoDiaz**
Superseeded the change proposed in #205 

@kf6kjg as it turned out, there was a problem with eslint so it was ignoring files on Unix systems.

Do you want to check this on Windows?

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -308,20 +308,20 @@ class WarmUp {
...
 const functions = ${JSON.stringify(functions)};
 
 module.exports.warmUp = async (event, context) => {
-  console.log("Warm Up Start");
+  console.log('Warm Up Start');
 
   const invokes = await Promise.all(functions.map(async (func) => {
     let concurrency;
-    const functionConcurrency = process.env["WARMUP_CONCURRENCY_" + func.name.toUpperCase().replace(/-/g, '_')]
+    const functionConcurrency = process.env[\`WARMUP_CONCURRENCY_${func.name.toUpperCase().replace(/-/g, '_')}\`]
```
Comment (kf6kjg) [inline src/index.js]: This line still fails ESLint with `error  'func' is not defined  no-undef`, could be bypassed by using the eslint comment `// eslint-disable-line no-undef`

File: test/hook.afterPackageCreateDeploymentArtifacts.test.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /* global jest beforeEach describe it expect */
 
+const path = require('path');
```
Comment (kf6kjg) [inline test/hook.afterPackageCreateDeploymentArtifacts.test.js]: ESLint notes `Expected 1 empty line after require statement not followed by another require  import/newline-after-import`

File: test/hook.afterPackageInitialize.test.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 /* global jest beforeEach describe it expect */
 
+const path = require('path');
```
Comment (kf6kjg) [inline test/hook.afterPackageInitialize.test.js]: ESLint notes `Expected 1 empty line after require statement not followed by another require  import/newline-after-import`

**Comment by juanjoDiaz** [comment]
Damn! I really messed those commit. It should be OK now 😅

**Comment by kf6kjg** [comment]
Nice, though I still have to deactivate the `linebreak-style` ESLint option to get things to pass checks: git on Windows, at least as I have it configured, checks out the source with CRLF and commits with LF.  This makes the repo linebreak format independent of the OS it's being edited on.

**Comment by juanjoDiaz** [comment]
> Nice, though I still have to deactivate the linebreak-style ESLint option to get things to pass checks: git on Windows, at least as I have it configured, checks out the source with CRLF and commits with LF. This makes the repo linebreak format independent of the OS it's being edited on.

Maybe your text editor is configured to change all the line endings to windows line endings?
You could configure it to use Unix line endings.

**Pull request merged**